dff/script/core/context.py

```diff
@@ -46,10 +47,26 @@ class Context(BaseModel):
     A structure that is used to store data about the context of a dialog.
     """
 
-    id: Union[UUID, int, str] = Field(default_factory=uuid4)
+    _storage_key: Optional[str] = PrivateAttr(default=None)
     """
-    `id` is the unique context identifier. By default, randomly generated using `uuid4` `id` is used.
-    `id` can be used to trace the user behavior, e.g while collecting the statistical data.
+    `_storage_key` is the unique private context identifier, by which it's stored in context storage.
+    By default, randomly generated using `uuid4` `_storage_key` is used.
+    `_storage_key` can be used to trace the user behavior, e.g while collecting the statistical data.
+    """
+    _primary_id: str = PrivateAttr(default_factory=lambda: str(uuid4()))
```

Comment:
Because we needed a simple and unique way to identify context in database.

1. Since now we require two IDs per context, we had to come up with different names for them.

2. Some databases can't use same fields to store ints and strings, so we decided to switch to strings (we can store ints as strings and parse them later if we need).